client/rpc/block_converter.go

```diff
@@ -0,0 +1,147 @@
+package rpc
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	cmn "github.com/tendermint/tendermint/libs/common"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	"github.com/tendermint/tendermint/types"
+	"github.com/tendermint/tendermint/version"
+	"sync"
+	"time"
+)
+
+type CosmosResultBlock struct {
+	BlockMeta CosmosBlockMeta `json:"block_meta"`
+	Block     CosmosBlock     `json:"block"`
+}
+
+type CosmosBlockMeta struct {
+	BlockID types.BlockID `json:"block_id"` // the block hash and partsethash
+	Header  CosmosHeader  `json:"header"`   // The block's Header
```

Comment:
```suggestion
	Header  types.Header  `json:"header"`   // The block's Header
```

this may better to not have to update